grpc/client/src/test/java/io/helidon/grpc/client/ProtoGrpcServiceClientIT.java

```diff
@@ -86,7 +86,7 @@ public static void startServer() throws Exception {
                 .get(10, TimeUnit.SECONDS);
 
         ClientServiceDescriptor descriptor = ClientServiceDescriptor
-                .builder(StringServiceGrpc.getServiceDescriptor())
+                .builder(StringServiceGrpc.class)
```

Comment:
This file is a bit of a muddle. There are test lifecycle methods (`@BeforeAll` etc...) at the top and at the bottom and then there are some at the bottom followed by another test method. Typically the lifecycle methods all go at the top followed by the tests.

Reply:
I had moved the methods for the demo and got accidentally checked in. Reverted back to the earlier version.

Reply:
Removed this file.